DOCS.md

```diff
@@ -887,6 +887,31 @@ __Arguments__
 
 ---------------------------------------
 
+<a name="setMessageReaction"></a>
+### api.setMessageReaction(reaction, messageID[, callback])
+
+Sets reaction on message
+
+__Arguments__
+
+* `reaction`: A string contains `emoji`, `emoji shortcut`, `emoji in unicode` or left `empty string` for delete reaction (look down for list of supported emojis).
```

Comment:
Can you change this to:
A string containing either an emoji, an emoji in unicode, an emoji shortcut (see list of supported emojis below).  The string can be left empty (`""`) in order to remove a reaction.